libyul/backends/wasm/TextTransform.cpp

```diff
@@ -56,7 +57,7 @@ string TextTransform::run(wasm::Module const& _module)
 	ret += "    (export \"main\" (func $main))\n";
 
 	for (auto const& g: _module.globals)
-		ret += "    (global $" + g.variableName + " (mut i64) (i64.const 0))\n";
+		ret += "    (global $" + g.variableName + " (mut " + g.type + ") (" + g.type + ".const 0))\n";
```

Comment:
Maybe we need to consider Whiskers in the future :)

Reply:
Should I add it to the refactor PR? Does not seem hard to do.